build/rollup.config.dev.js

```diff
@@ -15,6 +18,15 @@ export default [
     },
     plugins: [
       ...defaultPlugins,
+      alias({
+        entries: {
+          react: "preact/compat",
```

Comment:
We must alias `preact` to ensure material-ui imports `preact`, not `react`, when calling React-related functions. `preact` has a compatibility layer with an identical interface.